modules/bitgo/test/v2/unit/wallet.ts

```diff
@@ -1983,6 +1983,43 @@ describe('V2 Wallet:', function () {
         args[1]!.should.equal('full');
       });
 
+      it('should call prebuildTxWithIntent with the correct params for eth accelerations for receive address', async function () {
+        const recipients = [{
```

Comment:
Actually this testcase has a assertion that didn't expect recipients in the output because for intentType fillNonce and accleration we are not passing recipients in the method prebuildTxWithIntent

`args[0]!.should.not.have.property('recipients');`